.github/workflows/track-no-cover.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Check no cover Removal
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "15 9 1 * *"
+
+jobs:
+  check-no-cover:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.9"
+      - uses: extractions/setup-just@v1
+
+      - name: "Stop ignoring pragma: no cover lines"
+        run: |
+          set -eu
+          sed -i.bak 's/"pragma: no cover",/# "pragma: no cover",/' pyproject.toml
+
+      - name: Did coverage change?
+        env:
+          CI: 1
+        run: |
+          set -eu
+          set -o pipefail
+
+          just test | tee test-output.txt || true
+
+          # extract missing lines per file fom coverage report in the tests output
+          # python script: get the files table from coverage
+          # awk: break the table down into filename and missing lines number
+          python scripts/parse_coverage_output.py test-output.txt | awk '{ print $1 " " $3 }' - > no-cover.txt
```

Comment:
aside: this seems a little unstable against no-cover lines being moved around in a file, but looking at the contents of `no-cover.txt` a lot of them happen to be at the top of files so it doesn't seem worth worrying about.

Reply:
oh - ignore that - `missing lines number` means the count of missing lines, not a list of line numbers that are missing!

Reply:
Yep! I was originally just pulling out the whole coverage table which was far too unstable for this, my hope is count of lines is good enough 🤞